docker-compose.yml

```diff
@@ -5,7 +5,7 @@
 # DOCKER COMPOSE FOR FLOWKIT
 #
 
-version: '3.7'
+version: '3.5'
```

Comment:
Why do we need to revert this to an earlier version?

Reply:
We don't need any features of 3.7 format, only 3.5 which introduced `shm_size`. Let's us support a marginally broader range of docker releases.